pmlc/dialect/tile/ir/util.cc

```diff
@@ -75,6 +75,9 @@ unsigned typeScore(Type type) {
   if (type.isUnsignedInteger(32)) {
     return 10;
   }
+  if (type.isSignlessInteger(32)) {
```

Comment:
I think here we need to add SignlessInteger for 16, 32 and 64 bit width and return different results/number.

Reply:
fixed this.